Source/Urho3D/Container/Str.h

```diff
@@ -60,6 +60,15 @@ class URHO3D_API String
         *this = str;
     }
 
+    /// Move-construct from another string.
+    String(String && str) noexcept :
+        length_(0),
+        capacity_(0),
+        buffer_(&endZero)
+    {

```

Comment:
Is it necessary to initialize member variable here? as Swap also could assign them correct value.

Reply:
`Swap` will _swap_ them, not assign. And `String && str` definetely shan't be filled with uninitialized garbage.

Reply:
Makes sense. Thanks.